standard/expressions.md

```diff
@@ -1480,6 +1487,34 @@ parenthesized_expression
 
 A *parenthesized_expression* is evaluated by evaluating the *expression* within the parentheses. If the *expression* within the parentheses denotes a namespace or type, a compile-time error occurs. Otherwise, the result of the *parenthesized_expression* is the result of the evaluation of the contained *expression*.
 
+### §tuple-expressions-new-clause Tuple expressions
+
+A *tuple_expression* consists of two or more comma-separated and optionally-named *expression*s enclosed in parentheses.
+
+``` ANTLR
+tuple_expression
+    : `(` tuple_element (',' tuple_element)+ ')'
+tuple_element
+    : (identifier ':')? expression
+```
+
+A tuple expression is classified as a tuple. It has a type if all the element expressions have a type. In that case, the type of a tuple expression is the tuple type of the same arity, where each type element has the type - and name, where given - of the corresponding element expression.
+
+A tuple expression is evaluated by evaluating each of its element expressions in order from left to right.
+
+A tuple value can be obtained from a tuple expression by converting it to a tuple type (§implicit-tuple-conversions-new-clause), by reclassifying it as a value ([§11.2.2](expressions.md#1122-values-of-expressions))) or by making it the target of a deconstructing assignment (§deconstructing-assignment-new-clause).
+
+*Example:*
+
+``` c#
+(int i, string) t1 = (i: 1, "One");
+(int i, string) t2 = (i: 2, null);
+var t3 = (i: 3, "Three"); // (int i, string)
+var t4 = (i: 4, null); // Error: no type
```

Comment:
Would it be worth including an example where we end up with "necessary but odd-looking" parentheses, e.g.

```csharp
SomeMethod((1, 2));
```

(Obviously with more description!) It's the kind of thing I'd definitely mention in a tutorial - and I know the standard isn't a tutorial, but if we're already giving examples, perhaps one would be useful?

Reply:
Example added in upcoming commit